.github/workflows/gotests.yaml

```diff
@@ -64,19 +64,11 @@ jobs:
           restore-keys: |
             ${{ github.job }}-${{ runner.os }}-go-
 
-      - name: Install libgit2/git2go
-        run: make git2go
-
-      - name: Install Ginkgo
-        run: go install github.com/onsi/ginkgo/v2/ginkgo
-
-      - name: Install kraft
-        run: make kraft DOCKER= DISTDIR="$(go env GOPATH)"/bin
-
       - name: Run unit tests
-        run: ginkgo -v -p -randomize-all ./test/e2e/framework/...
+        run: go run github.com/onsi/ginkgo/v2/ginkgo -v -p -randomize-all ./test/e2e/framework/...
```

Comment:
I am a bit confused about this step actually, how does it differ from `Run e2e tests`?

Reply:
Those contain unit tests for the test framework itself.

The CLI tests are in test/cli.

I'm running the unit tests before the CLI tests to ensure that the framework's code is working as expected.

Reply:
Gotcha. Your latest update helped clarify things. Thanks!